mlflow/models/evaluation/default_evaluator.py

```diff
@@ -332,6 +347,95 @@ def __validate_artifacts(artifacts):
     )
 
 
+def _load_custom_metric_artifact(
+    dataset_name, temp_dir, artifact_name, raw_artifact, custom_metric_index, custom_metric_name
+):
+    if isinstance(raw_artifact, EvaluationArtifact):
+        return raw_artifact
+
+    exception_header = _get_custom_metric_exception_header(custom_metric_index, custom_metric_name)
+
+    # If path is str, convert to pathlib.Path. Then, type inference through file extension
+    if isinstance(raw_artifact, str):
+        raw_artifact = pathlib.Path(raw_artifact)
+    if isinstance(raw_artifact, pathlib.Path):
+        ext_to_artifact_map = {
+            ".png": ImageEvaluationArtifact,
+            ".jpg": ImageEvaluationArtifact,
```

Comment:
Should we support `.svg` format here for an additional `ImageEvaluationArtifact`? It's pretty widely used.

Reply:
`pillow` currently does not support `.svg` files as documented [here](https://pillow.readthedocs.io/en/latest/handbook/image-file-formats.html). We could extend support for this by adding additional dependencies like `cairosvg` or `svglib` to MLflow as suggested [here](https://stackoverflow.com/questions/15130670/pil-and-vectorbased-graphics). Should we add such dependencies?

Reply:
Perhaps not as part of this PR.